spec/features/groups_index_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+# rubocop:disable Metrics/BlockLength
+RSpec.feature 'Groups', type: :feature do
+  background do
+    visit root_path
+    click_link 'Sign up'
+    fill_in 'Name', with: 'Test'
+    fill_in 'Email', with: 'showanamalongo@gmail.com'
+    fill_in 'Password', with: 'password'
+    fill_in 'Password confirmation', with: 'password'
+    click_button 'Sign up'
+  end
+
+  scenario 'Welcome! You have signed up successfully.' do
+    expect(page).to have_content('Welcome! You have signed up successfully.')
+  end
+
+  scenario 'show empty transaction alert message' do
+    expect(page).to have_content('There is no category added!!!')
+  end
+
+  scenario 'add new category' do
+    click_link 'Add a new category'
+    fill_in 'Name', with: 'Test'
+    attach_file('Image', 'app/assets/images/Screen.png')
+    click_button 'Save'
+    expect(page).to have_content('Test')
+    expect(page).to have_button('Delete')
+  end
+
+  scenario 'go to created category' do
+    click_link 'Add a new category'
+    fill_in 'Name', with: 'Test'
+    attach_file('Image', 'app/assets/images/login.png')
+    click_button 'Save'
+    click_link 'Test'
+    expect(page).to have_content('Total Amount:')
+  end
+
+  scenario 'add new category' do
+    click_link 'Add a new category'
+    fill_in 'Name', with: 'Test'
+    attach_file('Image', 'app/assets/images/transactions_page.png')
+    click_button 'Save'
```

Comment:
- [ ] There are two tests are failing because of image not present. Please put the right name of the image file to pass the test.

![Screenshot from 2022-04-27 16-08-25](https://user-images.githubusercontent.com/26767383/165500572-e08e327b-7634-4040-9606-cf06da518dfb.png)

Reply:
Change Impemented.